src/store/messagesStore.js

```diff
@@ -1336,6 +1336,16 @@ const actions = {
 		message.token = targetToken
 		// Do not forward the message silently
 		message.silent = false
+		// Reset thread details
+		if (message.threadId) {
+			delete message.threadId
+		}
+		if (message.isThread) {
+			delete message.isThread
+			delete message.threadTitle
+			delete message.threadReplies
+		}
```

Comment:
message.threadId and message.isThread are always present. There should be no harm to remove them and other without any check conditions

```suggestion
		delete message.threadId
		delete message.isThread
		delete message.threadTitle
		delete message.threadReplies
```